TACC/Core-Portal
# task/FP-327 - Onboarding welcome messages

## Overview: ##

Add alert messages to each section
Message text edited by @thbrowntacc 

## PR Status: ##


## Related Jira tickets: ##

* [FP-327](https://jira.tacc.utexas.edu/browse/FP-327)

## Summary of Changes: ##

- Add alert message describing each section to Workbench component
- Welcome messages are permanently dismissable
- Welcome message dismissal state is saved in browser localStorage

## Testing Steps: ##
1. See and dismiss messages in each section
2. Reload the browser to make sure they stay gone
3. Reset your saved message state in Chrome by going to Dev Tools > "Application" Tab > Storage - Local Storage - https://cep.dev in the sidebar -- delete the welcomeMessages key

![image](https://user-images.githubusercontent.com/17181582/90830320-14b67700-e307-11ea-8dc1-3527cd6d4836.png)

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/90902189-b4b8e280-e391-11ea-8ed3-24aa981d8ef0.png)
![image](https://user-images.githubusercontent.com/17181582/90902203-baaec380-e391-11ea-94f0-4bd4a23c0eec.png)
![image](https://user-images.githubusercontent.com/17181582/90902213-bf737780-e391-11ea-85be-1a04dd132639.png)
![image](https://user-images.githubusercontent.com/17181582/90902225-c4382b80-e391-11ea-838c-dab96bdac85c.png)
![image](https://user-images.githubusercontent.com/17181582/90902237-ca2e0c80-e391-11ea-9cfa-76c32bcba161.png)


## Notes: ##
